### PR TITLE
force focus on button fix  #13146

### LINKED ISSUE #13146: v11.x - Accountancy plan : the new accountancy plan is not loaded after submit
# Instructions
# Bug
The javascript in charge to change the ''valid_change_chart' flag doesn't set it to 1 change it
File : accountancy/admin/account.php (line 105 and 275 to 292)

## Environment
- **Version**: 11.x
- **OS**: Darwin 14.5.0
- **Web server**: Apache/2.4.41 (Unix)
- **PHP**: 7.0.33
- **Database**: MYSQL 5.7.29
- **Browser**: firefox 72.0

### PR DESCRIPTION
**Pull request by joanlouis**
# Fix  #13146
[*Long description*]
force focus on button for Mac browsers to allow load of accounts

### CODE REVIEW

**Comment by hregis** [comment]
@joanlouis  @eldy thanks but same problem with Firefox for MAC OS !

**Comment by joanlouis** [comment]
> @joanlouis @eldy thanks but same problem with Firefox for MAC OS !

@hregis have you try my fix ? it work for me on safari 13.0.5 and firefox Mac 73.0.1

**Comment by hregis** [comment]
@joanlouis 
yes i try this fix with Firefox 73.0.1 in Mac OS X Catalina
if i try to perform a search, the "chart of accounts" is reload !

**Comment by joanlouis** [comment]
you're right I don't know why but the focus is always on mainbody when the form is submitted

**Comment by joanlouis** [comment]
i have change the javascript to change value on mousedown then it was not triggered on form submit and now it seem to work
@hregis can you confirm it work for you now ?

File: htdocs/accountancy/admin/account.php [hunk 1]
```diff
@@ -337,7 +330,7 @@
     else dol_print_error($db);
     print "</select>";
     print ajax_combobox("chartofaccounts");
-    print '<input type="submit" class="button" name="change_chart" id="change_chart" value="'.dol_escape_htmltag($langs->trans("ChangeAndLoad")).'">';
+    print '<input type="submit" onclick="change_chart.focus();change_chart.select();" class="button" name="change_chart" id="change_chart" value="'.dol_escape_htmltag($langs->trans("ChangeAndLoad")).'">';
```
Comment (hregis) [inline htdocs/accountancy/admin/account.php]: Maybe you can remove this?

**Comment by hregis** [comment]
@joanlouis if it's ok for you it's ok for me 😉👌

File: htdocs/accountancy/admin/account.php (hunk 1 above)
Comment (joanlouis) [reply htdocs/accountancy/admin/account.php]: yes this is a pb with git

**Comment by joanlouis** [comment]
now it is ok for me too

**Comment by eldy** [comment]
I pushed a different way to fix this because it was not working for bind people mode.
Can you check it is ok now ?

**Comment by joanlouis** [comment]
@eldy yes it is ok

**Pull request closed without merging**